pom.xml

```diff
@@ -50,7 +50,7 @@
         <dep.airlift.version>0.148</dep.airlift.version>
         <dep.packaging.version>${dep.airlift.version}</dep.packaging.version>
         <dep.slice.version>0.29</dep.slice.version>
-        <dep.aws-sdk.version>1.11.30</dep.aws-sdk.version>
+        <dep.aws-sdk.version>1.11.165</dep.aws-sdk.version>
```

Comment:
At Teradata we run basic tests against some static S3 tables so maybe I can run this PR on our internal Jenkins and let you know what happens. @smdorval is also currently working on adding support for creating/deleting S3 tables in Tempto which should enable us to write tests that are run on Travis.